Assets/HoloToolkit/Utilities/Scripts/Utils.cs

```diff
@@ -10,14 +10,10 @@ namespace HoloToolkit.Unity
     /// </summary>
     public static class Utils
     {
+        [System.Obsolete("Use GameObjectExtensions.SetLayerRecursively(gameObject, layer) instead;")]
```

Comment:
Awesome.  Does this PR also update the core MRTK scripts that use this obsolete reference?